### PR TITLE
arbitraries for json object

### PR DESCRIPTION
**Pull request by mikhailZakharevich**
* added Json Object generator from circe-testing

### CODE REVIEW

File: src/test/scala/group/scala/karazin/circe/literal/extras/generators/JsonGenerators.scala [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import org.scalacheck.Prop._
+import io.circe.Encoder.encodeJsonObject
+
+/**
+  * as an option, we can put all our json generation methods (incl. custom ones if need be)
+  * in a separate file, like here
+  */
+object JsonGenerators {
+
+  val genJsonObjectDefault: Gen[JsonObject] =
```
Comment (IgorWolkov) [inline src/test/scala/group/scala/karazin/circe/literal/extras/generators/JsonGenerators.scala]: For ScalaCheck we need given (aka implicit) arbitraries.
Please review the existing test.
What you need here is to add a single line:
```
given Arbitrary[JsonObject] = arbitraryJsonObject
```

Please review 
https://dotty.epfl.ch/docs/reference/contextual/givens.html
https://dotty.epfl.ch/docs/reference/contextual/using-clauses.html

File: src/test/scala/group/scala/karazin/circe/literal/extras/generators/JsonGenerators.scala [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import org.scalacheck.Gen
+import org.scalacheck.Arbitrary
+import org.scalacheck.Prop._
+import io.circe.Encoder.encodeJsonObject
+
+/**
+  * as an option, we can put all our json generation methods (incl. custom ones if need be)
+  * in a separate file, like here
+  */
+object JsonGenerators {
```
Comment (IgorWolkov) [inline src/test/scala/group/scala/karazin/circe/literal/extras/generators/JsonGenerators.scala]: Let's make it `trait` to be able to join all specific arbitraries.

Comment (IgorWolkov) [inline src/test/scala/group/scala/karazin/circe/literal/extras/generators/JsonGenerators.scala]: Let's rename it to `JsonArbitraries`.

**Review by IgorWolkov — CHANGES_REQUESTED** [review]
Improve generators

File: src/test/scala/group/scala/karazin/circe/literal/extras/generators/JsonGenerators.scala (hunk 2 above)
Comment (mikhailZakharevich) [reply src/test/scala/group/scala/karazin/circe/literal/extras/generators/JsonGenerators.scala]: Done

File: src/test/scala/group/scala/karazin/circe/literal/extras/generators/JsonGenerators.scala (hunk 1 above)
Comment (mikhailZakharevich) [reply src/test/scala/group/scala/karazin/circe/literal/extras/generators/JsonGenerators.scala]: If that's our approach, no problem.

File: src/test/scala/group/scala/karazin/circe/literal/extras/generators/JsonGenerators.scala (hunk 2 above)
Comment (mikhailZakharevich) [reply src/test/scala/group/scala/karazin/circe/literal/extras/generators/JsonGenerators.scala]: Done

**Review by IgorWolkov — APPROVED** [review]

**Pull request merged**